Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -5010,7 +5015,11 @@ void CGame::ProcessClientTriggeredEventSpam()
             if (GetTickCount64_() - data.m_llTicks >= m_iClientTriggeredEventsIntervalMs)
             {
                 if (data.m_uiCounter > m_iMaxClientTriggeredEventsPerInterval)
-                    player->CallEvent("onPlayerTriggerEventThreshold", {});
+                {
+                    CLuaArguments args;
+                    args.PushString(data.m_strLastEventName.c_str());
```

Comment:
`args.PushString(data.m_strLastEventName);` because this function has std::string argument overload.